### PR TITLE
Replaced v2.3 files with symlink to v2.2 file

### PR DESCRIPTION
**Pull request by meker12**
## Purpose of this pull request

This PR consolidates versioned topics in v2.2 and v2.3 to a single v2.2 topic, and replaces the v2.3 topic with a symlink to the v2.2 topic.

## Affected DevDocs pages

https://devdocs.magento.com/before/before-workspace-magento-prereqs.html
https://devdocs.magento.com/guides/v2.3/cloud/live/live-sanity-check.html
https://devdocs.magento.com/guides/v2.3/cloud/live/go-live-checklist.html

### CODE REVIEW

File: guides/v2.2/cloud/live/go-live-checklist.md [hunk 1]
```diff
@@ -6,9 +6,7 @@ functional_areas:
   - Testing
 ---
 
-## Pre-Launch Checklist {#checklist}
-
-Prior to launch and switching the DNS, you can download and follow the [Checklist]({{ site.baseurl }}/common/pdf/Pre-Launch.Checklist_4_23_2018.docx) and all tests for your deployed site or store. See an overview of the complete deployment process for Starter and Pro at [Deploy your store]({{ page.baseurl }}/cloud/live/stage-prod-live.html).
+Before you launch to Production, download the [Go live checklist]({{ site.baseurl }}/common/pdf/Pre-Launch.Checklist_4_23_2018.docx) and use it with these instructions to confirm that you have completed all required configuration and testing. See an overview of the complete deployment process for Starter and Pro at [Deploy your store]({{ page.baseurl }}/cloud/live/stage-prod-live.html).
```
Comment (hguthrie) [inline guides/v2.2/cloud/live/go-live-checklist.md]: The phrase "launch to Production" seems a little inaccurate. One would `deploy to the Production environment` and proceed through the launch steps, no?  Especially if it is the first time. May need clarification.

File: guides/v2.2/cloud/live/live-sanity-check.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ If you modified your `composer.json` file to add modules, we recommend running t
 Your Git branch must have the following files for building and deploying for your local and to Integration, Staging, and Production environments:
 
 * `auth.json` in the root Magento directory. This file includes the Magento authentication keys entered when creating the project. If you need to verify the file and settings, see [Troubleshoot deployment]({{ page.baseurl }}/cloud/trouble/troubleshoot-deployment.html).
-* `config.php` if you used [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html) for 2.2.X
+* `config.php` if you used [Configuration Management]({{ page.baseurl }}/cloud/live/sens-data-over.html)
```
Comment (hguthrie) [inline guides/v2.2/cloud/live/live-sanity-check.md]: This concept is old and only refers to upgrading from an older version of Magento. We may still need it, but I'm not 100% on that.

Comment (hguthrie) [reply guides/v2.2/cloud/live/live-sanity-check.md]: Here's some content about it. Could also provide link, maybe: https://devdocs.magedevteam.com/923/guides/v2.3/cloud/project/project-upgrade.html#configuration-management

**Comment by hguthrie** [comment]
Also, if this consolidates the `Install Magento prereqs` topic, I don't see any edits for that topic. There was a lot of version-specific language in it, like references to a PHP 7.0 requirement. Maybe I don't see what you did. Just checking.

**Comment by meker12** [comment]
@hguthrie This PR was purely to consolidate the topics to one.  I didn't do very much editing because I'm in the process or revising both topics.

File: guides/v2.2/cloud/live/go-live-checklist.md (hunk 1 above)
Comment (meker12) [inline guides/v2.2/cloud/live/go-live-checklist.md]:
```suggestion
Before you deploy to the Production environment, download the [Go live checklist]({{ site.baseurl }}/common/pdf/Pre-Launch.Checklist_4_23_2018.docx) and use it with these instructions to confirm that you have completed all required configuration and testing. See an overview of the complete deployment process for Starter and Pro at [Deploy your store]({{ page.baseurl }}/cloud/live/stage-prod-live.html).
```

**Comment by hguthrie** [comment]
> @hguthrie This PR was purely to consolidate the topics to one. I didn't do very much editing because I'm in the process or revising both topics.

I would remove it, then. It seems out of scope, especially if you can not ensure that the topic consistent and helpful for all versions.

**Comment by meker12** [comment]
> @hguthrie This PR was purely to consolidate the topics to one. I didn't do very much editing because I'm in the process or revising both topics.

Ah I see the issue-- I didn't commit the new file "prereqs" file.

File: guides/v2.2/cloud/before/before-workspace-magento-prereqs.md [hunk 3]
```diff
@@ -42,11 +42,11 @@ When using Vagrant, we also recommend the package [hostmanager](https://github.c
...
+We strongly recommend installing [Nginx]({{ page.baseurl }}/install-gde/prereq/nginx.html) for your web server on your local. While {{site.data.var.ee}} supports [Apache]({{ page.baseurl }}/install-gde/prereq/apache.html), {{site.data.var.ece}} uses Nginx. To have your local as close to cloud installations as possible, install and configure Nginx.
 
 ## PHP (local) {#php}
 
-Install [PHP](https://glossary.magento.com/php) on your local. We recommend PHP 7.0. For information on installing PHP, see these instructions for [CentOS]({{ page.baseurl }}/install-gde/prereq/php-centos.html) and [Ubuntu]({{ page.baseurl }}/install-gde/prereq/php-ubuntu.html). For instructions for another OS, see the [PHP documentation](http://php.net/manual/en/install.php).
+Install [PHP](https://glossary.magento.com/php) on your local. We recommend PHP 7.1.3+ or 7.2. For information on installing PHP on CentOS and Ubuntu see [PHP](../../../v2.3/install-gde/prereq/php-centos-ubuntu.html). For instructions for another OS, see the [PHP documentation](http://php.net/manual/en/install.php).
```
Comment (hguthrie) [inline guides/v2.2/cloud/before/before-workspace-magento-prereqs.md]: If you want a version-specific URL, you should put a `site.baseurl` here.

File: guides/v2.2/cloud/before/before-workspace-magento-prereqs.md [hunk 4]
```diff
@@ -58,7 +58,7 @@ The following packages may also be helpful for your PHP installation:
 * [intl](http://php.net/manual/en/book.intl.php)
 * [json](http://php.net/manual/en/ref.json.php)
 * [mbstring](http://php.net/manual/en/book.mbstring.php)
-* [mcrypt](http://php.net/manual/en/book.mcrypt.php)
+* [mcrypt](http://php.net/manual/en/book.mcrypt.php) (for PHP 7.1 and earlier only)
```
Comment (hguthrie) [inline guides/v2.2/cloud/before/before-workspace-magento-prereqs.md]: We don't support 7.1 anymore, so can probably remove this line.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**